tests/book.test.js

```diff
@@ -0,0 +1,161 @@
+/* tests/book.test.js */
+const { expect } = require('chai');
+const request = require('supertest');
+const { Book } = require('../src/models');
+const app = require('../src/app');
+
+describe('/books', () => {
+  before(async () => Book.sequelize.sync());
+
+  beforeEach(async () => {
+    await Book.destroy({ where: {} });
+  });
+
+  describe('with no records in the database', () => {
+    describe('POST /books', () => {
+      it('creates a new book in the database', async () => {
+        const response = await request(app).post('/books').send({
+          title: 'Brief Answers To The Big Questions',
+          // author: 'Stephen Hawking',
+          // genre: 'Autobiography',
+          ISBN: '9781473695993',
+        });
+
+        const newBookRecord = await Book.findByPk(response.body.id, {
+          raw: true,
+        });
+
+        expect(response.status).to.equal(201);
+        expect(response.body.title).to.equal(
+          'Brief Answers To The Big Questions'
+        );
+        // expect(newBookRecord.author).to.equal('Stephen Hawking');
+        // expect(newBookRecord.genre).to.equal('Autobiography');
+        expect(newBookRecord.ISBN).to.equal('9781473695993');
+      });
+
+      it('return 400 and validation error message if it does not pass the validation', async () => {
+        const response = await request(app).post('/books').send({
+          title: '',
+          author: '',
+        });
+
+        const newBook = await Book.findByPk(response.body.id, { raw: true });
+
+        expect(response.status).to.equal(400);
+        expect(response.body.errors.length).to.equal(1);
+        // expect(newBook).to.equal(null);
+      });
+    });
+  });
+});
+
+describe('with records in the database', () => {
+  beforeEach(async () => {
+    await Book.destroy({ where: {} });
+  });
+
+  let books;
+  beforeEach(async () => {
```

Comment:
Why all the commented out code here? As above this is not working and seems unfinished. This applies  to the rest of this file also, definitely needs looking at